bsubmit/README.md

```diff
@@ -0,0 +1,98 @@
+# bsubmit
+
+bsubmit is a wrapper of bsub. It allows users to submit jobs as another user.
+
+If you would like to contribute, you must follow the DCO process in the attached [DCO Readme file](https://github.com/IBMSpectrumComputing/lsf-utils/blob/master/IBMDCO.md) in the root of this repository.  It essentially requires you to provide a Sign Off line in the notes of your pull request stating that the work does not infinge on the work of others.  For more details, refer to the DCO Readme file.
+
+## Release Information
+
+* IBM Spectrum LSF bsubmit
+* Supporting LSF Release: 10.1
+* Wrapper Version: 1.0
```

Comment:
what does `Wrapper Version` mean? I guess it is `bsubmit version`.

Reply:
change to "Version"